README.md

```diff
@@ -15,30 +15,53 @@ This extension will allow you to access your **[zx2c4 pass](http://www.zx2c4.com
 You can choose to automatically fill and submit login forms if a matching password entry is found.
 
 ### Installation
-- You need **[zx2c4 pass](http://www.zx2c4.com/projects/password-store/)** installed on your computer and a password respository with [password files configured to our format](#password-configuration)
-- You will also need to **[install the host application](docs/INSTALLATION.md#installing-the-host-application)** so the extension can communicate with `pass` to retrieve your passwords
-- Install the current release for your browser:
+
+##### xz2c4 pass repository
+This extension requires **[zx2c4 pass](http://www.zx2c4.com/projects/password-store/)** to be installed and set up with a password repository. Make sure you can execute `pass show some-password-name` in a terminal before continuing.
+
+##### Host application
+For the extension to communicate with your system's `pass` script, you need to install what's called the host application from [the official GitHub repository](https://github.com/passff/passff-host).
+The host application allows the extension to communicate with `pass` on your system.
+
+##### PassFF extension
+Install the current release of PassFF for your browser:
   - [Firefox](https://addons.mozilla.org/firefox/addon/passff)
-  - Chrome, Chromium (coming soon)
-  - Vivaldi, Opera (coming soon)
+  - Chrome, Chromium, Opera, Vivaldi: [Pending Chrome port](https://github.com/passff/passff/issues/105)
```

Comment:
.... why do you decide not to mention it in the introduction but mention it here? I suggest removing all of the references to other browsers from the documentation as long as we don't have compatibility with any other browser but Firefox.

Reply:
I'd like users to know about the pending Chrome port, so as to foster interest & contribution!
I moved the mention at the top of the page. However, I'm not against removing it entirely.